firmware/application/usb_serial_asyncmsg.hpp

```diff
@@ -41,136 +44,4 @@ class UsbSerialAsyncmsg {
     static void asyncmsg(const char* data);  // string literal
 };
 
-/*Notes:
- * - Don't use MayhemHub since it currently not support real time serial output
```

Comment:
would it be possible that we can still keep this comment?
but it's your decision